### PR TITLE
Add support for Accountant role APIs

### PR DESCRIPTION
**Pull request by ashwin1111**

### CODE REVIEW

**Comment by ashwin1111** [comment]
@Shwetabhk please review, I will be merging in this PR once the APIs are on live on staging

**Comment by ashwin1111** [comment]
will re run tests once apis are live

File: test/integration/accountant/fixtures/mock_accountant_data.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "get_orgs": [
+    {
+      "name": "Goldman Sachs",
+      "id": "orgs124",
+      "logo_download_url": "https://xyz.something.something",
+      "pending_invitations": 14,
+      "active_users_count": 34,
```
Comment (Shwetabhk) [inline test/integration/accountant/fixtures/mock_accountant_data.json:8]: `billed_users_count`

**Comment by Shwetabhk** [comment]
Approved, please merge only once the API is live and all tests pass @ashwin1111

**Comment by Shwetabhk** [comment]
Also do 1 cleanup after that

**Pull request merged**